packages/server/src/queue/queue-clean/queue-clean.service.spec.ts

```diff
@@ -90,72 +61,8 @@ describe('QueueService', () => {
       expect(question.status).toEqual('Stale');
     });
 
-    it('if no staff are present all questions with open status are marked as stale', async () => {
-      const ofs = await ClosedOfficeHourFactory.create();
```

Comment:
It seems like this will also probably require some changes to the QueueClean service itself - I'm pretty sure what we settled on was that we still want to regularly clean queues based on the cron time, but we're not gonna pop the clean queue modal when a TA checks out at the end of the day. 

It looks like we don't need the service method `shouldCleanQueue()` anymore, and we don't need to return the corresponding flag for it in `courseController.checkOut()`. And then we can delete the modal and stuff on the frontend. 

The other change to the queue clean service is that `checkoutAllStaff()` [shouldn't check `queue.areThereOfficeHoursRightNow()`](https://github.com/sandboxnu/office-hours/blob/d6435da929c901d86b0f41785f5b82be4fa2a70d/packages/server/src/queue/queue-clean/queue-clean.service.ts#L52). You can't check `queue.checkIsOpen()` because that just checks if there are any staff checked in, and then this clean function will never checkout any staff at all. I guess we can just get rid of the conditional and always clean the queue every day at 3am, which is maybe bad for courses if they are international or have weird schedules?? @dfarooq610 let me know what you think I suppose we could figure out some way to configure a clean time per course and profs can choose when they want the cleaning to happen...?

Reply:
also probably worth discussing what parts of this work should be added to this PR vs what should be an additional issue i am open to whatever ppl think

`configure a clean time per course` is definitely a feature that we can defer tho

Reply:
Saw the `checkoutAllStaff()` behavior was updated, so I made an issue to resolve the `shouldCleanQueue()` stuff in a later PR since that will involve frontend as well: https://github.com/sandboxnu/office-hours/issues/840

Reply:
I think now that OH are primarily based in the US, we do not have to worry about a long queue at 3AM (or whatever cron time we choose)

But yea removing the modal sounds good to me